bundle/bundle.go

```diff
@@ -43,6 +45,33 @@ type Bundle struct {
 	AutoApprove bool
 }
 
+const ExtraIncludePathsKey string = "DATABRICKS_BUNDLE_INCLUDE_PATHS"
+
+// Get extra include paths from environment variable
+func GetExtraIncludePaths() []string {
+	value, exists := os.LookupEnv(ExtraIncludePathsKey)
+	if !exists {
+		return []string{}
+	}
+	var sep = ":"
+	if runtime.GOOS == "windows" {
+		sep = ";"
+	}
+	return strings.Split(value, sep)
+}
+
+// Try loading bundle config from one of the extra include paths from the environment variable
+// If not able to load, do not return an error, just return nil
+func tryLoadFromExtraIncludePaths() *Bundle {
+	for _, extraIncludePath := range GetExtraIncludePaths() {
+		bundle := &Bundle{}
+		if err := bundle.Config.Load(extraIncludePath); err == nil {
```

Comment:
This seems to imply that now this path is bundle root path. Can't we just use `DATABRICKS_BUNDLE_ROOT` instead?

Reply:
To clarify: calling `bundle.Config.Load` changes configured bundle root